docs/standard-input-output.adoc

```diff
@@ -0,0 +1,64 @@
+:page-title: Standard input/output
+:page-description: JUnit Jupiter extension for simulating standard input or catching standard output.
+
+The `StdIOExtension` adds a simple way to test your classes that read from the standard
+input (`System.in`) or write to the standard output (`System.out`).
+
+== How to use the extension
+
+If you would like to use the special parameters provided by the extension
```

Comment:
i think we do favor one sentence per line in our documentation, as it is easier to maintain and to move around (no line length). @nicolaiparlog explained the reason in the last pioneer stream, i think he know all the words, and has the best words, so hopefully he can explain that better in written words.

so i think we should stick to one sentence per line, without breaking.

Reply:
Yes I agree. The ~70 character line with manual wrap is only for commit messages, but not for .adoc-files.